### DISCUSSION TITLE
HTTP 401 Unauthorised

### DISCUSSION THREAD

**Discussion by Tamariniak** [bitfireAT/icsx5#52] [Q&A]
I have a link to a calendar in the following format:
`https://<domain>.<TLD>:443/calendars/user/<username>?accessToken=<access token>`
Which gives me a 401 error when I try to add it into ICSx5.

I used to be able to sync this link with ICSx5 before, but I started getting 404 errors a few weeks ago. I ended up reinstalling the app and now i get 401s when I try to add the calendar back.

I have ICSx5 installed from F-Droid.

Am I doing something wrong? I have the link synced with Nextcloud, but for some reason none of the events actually show up there. Same goes for when I tried adding it into Google Calendar (where it may just not be synced yet since I only added it a few moments ago to see whether authentication would fail). It has always worked with ICSx5 though.

I have also tried selecting "Requires authentication" and putting in my credentials for the site, but it didn't change anything (which it shouldn't as per the site's documentation). I also tried using the acces token instead of the password.

I have also tried linking the Nextcloud calendar that I otherwise sync using DAVx5, but had no luck that way either.

EDIT: I also think it is worth mentioning that DAVx5 doesn't recognise ICSx5 as being installed and gives me a "No webcal sync app found: install ICSx5" when I try to sync the calendar in DAVx5, even though both apps are installed (both from F-Droid).

EDIT2: I have also posted about this to Nextcloud's subreddit [here](https://www.reddit.com/r/NextCloud/comments/umrw4j/401_unauthorised_when_trying_to_sync_a_webcal/).

**Comment by rfc2822** [comment]
Sounds like the calendar source has some problems/downtimes etc. Does it work when you access the URL from the browser (at different times)?

Where is this calendar from?

**Reply by EngineerIgnazium** [reply]
```
``
```

**Reply by Tamariniak** [reply]
Started working recently, looks like the issue was on the side of the calendar's maintainer. I know that accessing the domain name would work, but I never thought to try accessing the whole calendar URL from a browser. Thanks for getting back to me!

**Comment by EngineerIgnazium** [comment]
![Uploading desktop-icon.svg…]()

**Comment by EngineerIgnazium** [comment]
This is not your average build!
